Makefile-tests.am

```diff
@@ -234,12 +234,7 @@ EXTRA_DIST += \
 	tests/libostreetest.h \
 	tests/libtest.sh \
 	tests/gpg-verify-data/README.md \
-	tests/gpg-verify-data/lgpl2 \
-	tests/gpg-verify-data/lgpl2.sig \
-	tests/gpg-verify-data/pubring.gpg \
-	tests/gpg-verify-data/secring.gpg \
-	tests/gpg-verify-data/trustdb.gpg \
-	tests/gpg-verify-data/gpg.conf
+	$(NULL)
```

Comment:
The commit message for this is wrong now that the meat of it is handled by 70a11189559856ec5aabed5ee8c62d0792a19ca8. Now it's just removing files in EXTRA_DIST that duplicate dist_gpgvinsttest_DATA.